python/lsst/daf/butler/registry/interfaces/_dimensions.py

```diff
@@ -25,7 +25,7 @@
 from abc import ABC, abstractmethod
 from typing import Optional
 
-from sqlalchemy.sql import FromClause
+import sqlalchemy
```

Comment:
seems like overkill to import everything. Why was this changed? and or why not just import sqlachemy.sql

Reply:
I've come to the conclusion that SQLAlchemy isn't well suited to `from <module> import <symbol>`, because so many of its symbols are tiny and don't provide enough context - things like `Sequence`, `Table`, `and_` etc.  More musings on this subject in a post from a few days ago [on slack](https://lsstc.slack.com/archives/C2JPL8U78/p1579878585048400).

And `import sqlalchemy.sql` would still invoke `sqlalchemy/__init__.py` - the only reason to use it would be to force the `sql` subpackage to also be imported, but SQLAlchemy is designed for that not to be necessary.